src/bitcoind_rpc.rs

```diff
@@ -359,6 +385,28 @@ pub enum ScanProgress {
     Progress { duration: u32, progress: f64 },
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq)]
+pub struct Unspent {
+    #[serde(rename = "txid")]
+    pub tx_id: Txid,
+    pub vout: u32,
+    pub address: Address,
+    pub label: String,
+    #[serde(rename = "scriptPubKey")]
+    pub script_pub_key: String,
+    pub amount: f64,
```

Comment:
Life would be made easier if this was deserialized directly in `::bitcoin::Amount`.